acceptance/redis32/main.go

```diff
@@ -22,16 +23,44 @@ func writeError(w http.ResponseWriter, err error) {
 	message, _ := json.Marshal(map[string]string{
 		"error": err.Error(),
 	})
-	w.Write(message)
 	w.Header().Set("Content-Type", "application/json")
 	w.WriteHeader(http.StatusInternalServerError)
+	w.Write(message)
+}
+
+func newPool(addr string, password string) *redis.Pool {
```

Comment:
back to my last comment on it, _I think_ it's okay to revert this back to using `redis.Dial` but I'll need to create the connection at the beginning of all the handlers rather than inside of `main()`.

I focused on pools without giving it much thought, and this is a bit better when the masters change because of the connection timeouts, etc.